### ISSUE TITLE
Multiple transformers broken

### ISSUE THREAD

**Issue by sebastianmarkow** [paulgb/sklearn-pandas#38]
Multiple transformers seems to be broken (for me at least).

Sample from the documentation:
~~~python
import sklearn as sk
import pandas as pd
import numpy as np
from sklearn_pandas import DataFrameMapper
    
mapper3 = DataFrameMapper([
    (['age'], [sk.preprocessing.Imputer(),
               sk.preprocessing.StandardScaler()])])
data_3 = pd.DataFrame({'age': [1, np.nan, 3]})
mapper3.fit_transform(data_3)
~~~

Issue:

~~~

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-8-b8b93bde93f1> in <module>()
      8                sk.preprocessing.StandardScaler()])])
      9 data_3 = pd.DataFrame({'age': [1, np.nan, 3]})
---> 10 mapper3.fit_transform(data_3)

/usr/local/lib/python2.7/site-packages/sklearn/base.pyc in fit_transform(self, X, y, **fit_params)
    431         if y is None:
    432             # fit method of arity 1 (unsupervised transformation)
--> 433             return self.fit(X, **fit_params).transform(X)
    434         else:
    435             # fit method of arity 2 (supervised transformation)

/usr/local/lib/python2.7/site-packages/sklearn_pandas/__init__.pyc in fit(self, X, y)
    111         for columns, transformer in self.features:
    112             if transformer is not None:
--> 113                 transformer.fit(self._get_col_subset(X, columns))
    114         return self
    115 

AttributeError: 'list' object has no attribute 'fit'
~~~

**Comment by dukebody** [comment]
You are using an old version of the code. Please update with pip -U or use an editable git source in requirements.txt.

**Comment by sebastianmarkow** [comment]
Seems like I'm on the latest 0.0.10
Am I missing something?

~~~
(1) ~/r/g/s/san-francisco-crime-kaggle (master|✔) insert $ pip2 install --upgrade sklearn-pandas
Requirement already up-to-date: sklearn-pandas in /usr/local/lib/python2.7/site-packages
Collecting numpy>=1.6.1 (from sklearn-pandas)
  Using cached numpy-1.9.3-cp27-none-macosx_10_6_intel.macosx_10_9_intel.macosx_10_9_x86_64.macosx_10_10_intel.macosx_10_10_x86_64.whl
Requirement already up-to-date: pandas>=0.11.0 in /usr/local/lib/python2.7/site-packages (from sklearn-pandas)
Requirement already up-to-date: scikit-learn>=0.13 in /usr/local/lib/python2.7/site-packages (from sklearn-pandas)
Collecting pytz>=2011k (from pandas>=0.11.0->sklearn-pandas)
  Using cached pytz-2015.6-py2.py3-none-any.whl
Collecting python-dateutil (from pandas>=0.11.0->sklearn-pandas)
  Downloading python_dateutil-2.4.2-py2.py3-none-any.whl (188kB)
    100% |████████████████████████████████| 192kB 1.3MB/s
Collecting six>=1.5 (from python-dateutil->pandas>=0.11.0->sklearn-pandas)
  Downloading six-1.9.0-py2.py3-none-any.whl
Installing collected packages: numpy, pytz, six, python-dateutil
  Found existing installation: numpy 1.9.1
    Uninstalling numpy-1.9.1:
      Successfully uninstalled numpy-1.9.1
  Found existing installation: pytz 2015.2
    Uninstalling pytz-2015.2:
      Successfully uninstalled pytz-2015.2
  Found existing installation: six 1.8.0
    Uninstalling six-1.8.0:
      Successfully uninstalled six-1.8.0
  Found existing installation: python-dateutil 2.4.1
    Uninstalling python-dateutil-2.4.1:
      Successfully uninstalled python-dateutil-2.4.1
Successfully installed numpy-1.9.3 python-dateutil-2.4.2 pytz-2015.6 six-1.9.0
~/r/g/s/san-francisco-crime-kaggle (master|✔) insert $ python2
Python 2.7.10 (default, Sep 14 2015, 00:10:17)
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.72)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy
>>> import sklearn_pandas
>>> sklearn_pandas.__version__
'0.0.10'
>>>
~~~

**Comment by dukebody** [comment]
Yes, the docs you see on github are for the latest master version, which doesn't have a release yet on PyPI. If you want to install the latest master version point pip to the git repository:

    pip2 install --upgrade git+https://github.com/paulgb/sklearn-pandas.git@master

**Comment by sebastianmarkow** [comment]
I see, well that triggered the confusion. 

Thanks Isreal

**Comment by dukebody** [comment]
We've released version 0.0.12 in PyPI, 0.0.12, that allow list transformers. So you don't have to install from github anymore.
